web-app/src/app/components/Footer.tsx

```diff
@@ -14,6 +14,20 @@ const Footer: React.FC = () => {
 
   return (
     <footer className='footer'>
+      <a
+        href={'https://share.mobilitydata.org/validator-feedback'}
```

Comment:
Link should be updated: https://share.mobilitydata.org/mobility-database-feedback

Reply:
done!